ogr/ogrsf_frmts/generic/makefile.vc

```diff
@@ -17,7 +17,7 @@ BASEFORMATS = -DSHAPE_ENABLED -DMITAB_ENABLED -DNTF_ENABLED -DSDTS_ENABLED -DTIG
 
 EXTRAFLAGS =	-I.. -I..\.. $(OGDIDEF) $(FMEDEF) $(OCIDEF) $(PGDEF) \
 		$(ODBCDEF) $(SQLITEDEF) $(MYSQLDEF) $(ILIDEF) $(DWGDEF) \
-		$(BASEFORMATS) $(IDBDEF) $(NASDEF) $(DODSDEF) \
+		$(BASEFORMATS) $(IDBDEF) $(NASDEF) $(DODSDEF) $(HANADEF) \
```

Comment:
the changes in that file should be reverted

Reply:
Done in 9655cc48c53c137355403d91e727ffa0539d75de.